### PR TITLE
Adding wait time for PV and pod status check in test case

### PR DESCRIPTION
**Pull request by jilju**
* Added wait time for verifying PV deletion
* Added sufficient timeout for second pod to be in Running state after deleting first pod

Signed-off-by: Jilju Joy <jijoy@redhat.com>

### CODE REVIEW

**Comment by jilju** [comment]
log:
```
Collecting Cluster versions
============================================================================================================ test session starts =============================================================================================================
platform linux -- Python 3.7.3, pytest-5.0.1, py-1.8.0, pluggy-0.12.0
rootdir: /home/jijoy/OCS-CI-Working/ocs-ci, inifile: pytest.ini
plugins: metadata-1.8.0, html-1.21.1, reportportal-1.0.5, logger-0.5.1, marker-bugzilla-0.9.1.dev2
collected 1 item                                                                                                                                                                                                                             

tests/manage/pv_services/test_verify_rbd_based_rwo_pvc.py::TestRbdBasedRwoPvc::test_rbd_based_rwo_pvc[Delete] 
--------------------------------------------------------------------------------------------------------------- live log setup ---------------------------------------------------------------------------------------------------------------
11:49:16 - MainThread - tests.conftest - INFO - All logs located at /tmp/ocs-ci-logs-1564640338
11:49:16 - MainThread - /home/jijoy/OCS-CI-Working/ocs-ci/ocs_ci/deployment/factory.py - INFO - Deployment key = aws_ipi
11:49:16 - MainThread - /home/jijoy/OCS-CI-Working/ocs-ci/ocs_ci/deployment/factory.py - INFO - Current deployment platform: AWS,deployment type: ipi
11:49:16 - MainThread - ocs_ci.utility.utils - INFO - Executing command: ./bin/openshift-install version
11:49:16 - MainThread - ocs_ci.utility.utils - INFO - OpenShift Installer version: ./bin/openshift-install v4.1.1-201906040019-dirty
                                                      built from commit fb776038a1d90b2b83839ab5deb8579287972e11
                                                      release image quay.io/openshift-release-dev/ocp-release@sha256:9c5f0df8b192a0d7b46cd5f6a4da2289c155fd5302dec7954f8f06c878160b8b
11:49:16 - MainThread - ocs_ci.utility.utils - INFO - Executing command: ./bin/oc version
11:49:22 - MainThread - ocs_ci.utility.utils - INFO - OpenShift Client version: Client Version: version.Info{Major:"4", Minor:"1+", GitVersion:"v4.1.0-201905091432+4910781-dirty", GitCommit:"4910781", GitTreeState:"dirty", BuildDate:"2019-05-09T19:19:42Z", GoVersion:"go1.11.5", Compiler:"gc", Platform:"linux/amd64"}
                                                      Server Version: version.Info{Major:"1", Minor:"13+", GitVersion:"v1.13.4+c62ce01", GitCommit:"c62ce01", GitTreeState:"clean", BuildDate:"2019-06-27T18:14:14Z", GoVersion:"go1.11.6", Compiler:"gc", Platform:"linux/amd64"}
11:49:22 - MainThread - ocs_ci.ocs.openshift_ops - INFO - Testing access to cluster with /home/jijoy/jijoy-ocs-deployment/auth/kubeconfig
11:49:22 - MainThread - ocs_ci.utility.utils - INFO - Executing command: oc cluster-info
11:49:24 - MainThread - ocs_ci.ocs.openshift_ops - INFO - Access to cluster is OK!
11:49:24 - MainThread - /home/jijoy/OCS-CI-Working/ocs-ci/ocs_ci/deployment/aws.py - WARNING - OCP cluster is already running, skipping installation
11:49:24 - MainThread - ocs_ci.utility.utils - INFO - Executing command: oc -n rook-ceph --kubeconfig /home/jijoy/jijoy-ocs-deployment/auth/kubeconfig get CephCluster  -o yaml
11:49:25 - MainThread - /home/jijoy/OCS-CI-Working/ocs-ci/ocs_ci/deployment/deployment.py - WARNING - OCS cluster already exists
11:49:25 - Dummy-1 - ocs_ci.utility.utils - INFO - Executing command: oc --kubeconfig /home/jijoy/jijoy-ocs-deployment/auth/kubeconfig get Pod -A -o yaml
11:49:25 - Dummy-2 - ocs_ci.utility.utils - INFO - Executing command: oc --kubeconfig /home/jijoy/jijoy-ocs-deployment/auth/kubeconfig get StorageClass -A -o yaml
11:49:25 - Dummy-3 - ocs_ci.utility.utils - INFO - Executing command: oc --kubeconfig /home/jijoy/jijoy-ocs-deployment/auth/kubeconfig get CephFileSystem -A -o yaml
11:49:25 - Dummy-4 - ocs_ci.utility.utils - INFO - Executing command: oc --kubeconfig /home/jijoy/jijoy-ocs-deployment/auth/kubeconfig get CephBlockPool -A -o yaml
11:49:25 - Dummy-5 - ocs_ci.utility.utils - INFO - Executing command: oc --kubeconfig /home/jijoy/jijoy-ocs-deployment/auth/kubeconfig get PersistentVolume -A -o yaml
11:49:25 - Dummy-6 - ocs_ci.utility.utils - INFO - Executing command: oc --kubeconfig /home/jijoy/jijoy-ocs-deployment/auth/kubeconfig get PersistentVolumeClaim -A -o yaml
11:49:25 - Dummy-7 - ocs_ci.utility.utils - INFO - Executing command: oc --kubeconfig /home/jijoy/jijoy-ocs-deployment/auth/kubeconfig get Secret -A -o yaml
11:49:25 - Dummy-8 - ocs_ci.utility.utils - INFO - Executing command: oc --kubeconfig /home/jijoy/jijoy-ocs-deployment/auth/kubeconfig get Namespace -A -o yaml
11:49:59 - MainThread - ocs_ci.utility.utils - INFO - Executing command: oc -n rook-ceph --kubeconfig /home/jijoy/jijoy-ocs-deployment/auth/kubeconfig get Pod --selector=app=rook-ceph-tools -o yaml
11:50:01 - MainThread - ocs_ci.utility.utils - INFO - Executing command: oc -n rook-ceph --kubeconfig /home/jijoy/jijoy-ocs-deployment/auth/kubeconfig rsh rook-ceph-tools-76c7d559b6-xn8tc ceph auth get-key client.admin --format json-pretty
11:50:05 - MainThread - ocs_ci.ocs.resources.ocs - INFO - Adding Secret with name secret-test-rbd-0111500557
11:50:05 - MainThread - ocs_ci.utility.utils - INFO - Executing command: oc -n rook-ceph --kubeconfig /home/jijoy/jijoy-ocs-deployment/auth/kubeconfig create -f /tmp/Secretad0e45a_ -o yaml
11:50:06 - MainThread - ocs_ci.ocs.resources.ocs - INFO - Adding CephBlockPool with name cbp-test-0111500672
11:50:06 - MainThread - ocs_ci.utility.utils - INFO - Executing command: oc -n rook-ceph --kubeconfig /home/jijoy/jijoy-ocs-deployment/auth/kubeconfig create -f /tmp/CephBlockPoolby4lgbci -o yaml
11:50:07 - MainThread - ocs_ci.utility.utils - INFO - Executing command: oc -n rook-ceph --kubeconfig /home/jijoy/jijoy-ocs-deployment/auth/kubeconfig get CephBlockPool cbp-test-0111500672 -o yaml
11:50:09 - MainThread - tests.helpers - INFO - Verifying that block pool cbp-test-0111500672 exists
11:50:09 - MainThread - ocs_ci.utility.utils - INFO - Executing command: oc -n rook-ceph --kubeconfig /home/jijoy/jijoy-ocs-deployment/auth/kubeconfig get Pod --selector=app=rook-ceph-tools -o yaml
11:50:10 - MainThread - ocs_ci.utility.utils - INFO - Executing command: oc -n rook-ceph --kubeconfig /home/jijoy/jijoy-ocs-deployment/auth/kubeconfig rsh rook-ceph-tools-76c7d559b6-xn8tc ceph osd lspools --format json-pretty
11:50:13 - MainThread - tests.helpers - INFO - POOLS are [{'poolnum': 1, 'poolname': 'rbd'}, {'poolnum': 2, 'poolname': 'ocsci-cephfs-metadata'}, {'poolnum': 3, 'poolname': 'ocsci-cephfs-data0'}, {'poolnum': 45, 'poolname': 'cbp-test-3121480953'}, {'poolnum': 46, 'poolname': 'cbp-test-3122012256'}, {'poolnum': 87, 'poolname': 'cbp-test-0111500672'}]
11:50:13 - MainThread - ocs_ci.utility.utils - INFO - Executing command: oc new-project namespace-test-0111501388
--------------------------------------------------------------------------------------------------------------- live log call ----------------------------------------------------------------------------------------------------------------
11:50:15 - MainThread - ocs_ci.ocs.resources.ocs - INFO - Adding StorageClass with name storageclass-test-rbd-0111501599
11:50:15 - MainThread - ocs_ci.utility.utils - INFO - Executing command: oc -n rook-ceph --kubeconfig /home/jijoy/jijoy-ocs-deployment/auth/kubeconfig create -f /tmp/StorageClassxeygcxxy -o yaml
11:50:17 - MainThread - ocs_ci.ocs.resources.ocs - INFO - Adding PersistentVolumeClaim with name pvc-test-0111501720
11:50:17 - MainThread - ocs_ci.utility.utils - INFO - Executing command: oc -n namespace-test-0111501388 --kubeconfig /home/jijoy/jijoy-ocs-deployment/auth/kubeconfig create -f /tmp/PersistentVolumeClaimhkw1t2h0 -o yaml
11:50:18 - MainThread - ocs_ci.utility.utils - INFO - Executing command: oc -n namespace-test-0111501388 --kubeconfig /home/jijoy/jijoy-ocs-deployment/auth/kubeconfig get PersistentVolumeClaim pvc-test-0111501720 -o yaml
11:50:19 - MainThread - tests.manage.pv_services.test_verify_rbd_based_rwo_pvc - INFO - Creating two pods which use PVC pvc-test-0111501720
11:50:19 - MainThread - ocs_ci.ocs.resources.ocs - INFO - Adding Pod with name pod-test-0111501948
11:50:19 - MainThread - ocs_ci.utility.utils - INFO - Executing command: oc -n namespace-test-0111501388 --kubeconfig /home/jijoy/jijoy-ocs-deployment/auth/kubeconfig create -f /tmp/POD_2sn3d_fh -o yaml
11:50:20 - MainThread - ocs_ci.utility.utils - INFO - Executing command: oc -n namespace-test-0111501388 --kubeconfig /home/jijoy/jijoy-ocs-deployment/auth/kubeconfig get Pod pod-test-0111501948 -o yaml
11:50:21 - MainThread - ocs_ci.utility.utils - INFO - Executing command: oc -n namespace-test-0111501388 --kubeconfig /home/jijoy/jijoy-ocs-deployment/auth/kubeconfig get Pod pod-test-0111501948 -o yaml
11:50:26 - MainThread - ocs_ci.utility.utils - INFO - Executing command: oc -n namespace-test-0111501388 --kubeconfig /home/jijoy/jijoy-ocs-deployment/auth/kubeconfig get Pod pod-test-0111501948 -o yaml
11:50:30 - MainThread - ocs_ci.utility.utils - INFO - Executing command: oc -n namespace-test-0111501388 --kubeconfig /home/jijoy/jijoy-ocs-deployment/auth/kubeconfig get Pod pod-test-0111501948 -o yaml
11:50:34 - MainThread - ocs_ci.utility.utils - INFO - Executing command: oc -n namespace-test-0111501388 --kubeconfig /home/jijoy/jijoy-ocs-deployment/auth/kubeconfig get Pod pod-test-0111501948 -o yaml
11:50:38 - MainThread - ocs_ci.utility.utils - INFO - Executing command: oc -n namespace-test-0111501388 --kubeconfig /home/jijoy/jijoy-ocs-deployment/auth/kubeconfig get Pod pod-test-0111501948 -o yaml
11:50:42 - MainThread - ocs_ci.utility.utils - INFO - Executing command: oc -n namespace-test-0111501388 --kubeconfig /home/jijoy/jijoy-ocs-deployment/auth/kubeconfig get Pod pod-test-0111501948 -o yaml
11:50:44 - MainThread - tests.helpers - INFO - Pod pod-test-0111501948 reached state Running
11:50:44 - MainThread - ocs_ci.utility.utils - INFO - Executing command: oc -n namespace-test-0111501388 --kubeconfig /home/jijoy/jijoy-ocs-deployment/auth/kubeconfig get Pod pod-test-0111501948 -o yaml
11:50:45 - MainThread - ocs_ci.ocs.resources.ocs - INFO - Adding Pod with name pod-test-0111504528
11:50:45 - MainThread - ocs_ci.utility.utils - INFO - Executing command: oc -n namespace-test-0111501388 --kubeconfig /home/jijoy/jijoy-ocs-deployment/auth/kubeconfig create -f /tmp/POD_pid0q8hu -o yaml
11:50:46 - MainThread - ocs_ci.utility.utils - INFO - Executing command: oc -n namespace-test-0111501388 --kubeconfig /home/jijoy/jijoy-ocs-deployment/auth/kubeconfig get Pod pod-test-0111504528 -o yaml
11:50:47 - MainThread - ocs_ci.utility.utils - INFO - Executing command: oc -n namespace-test-0111501388 --kubeconfig /home/jijoy/jijoy-ocs-deployment/auth/kubeconfig get Pod pod-test-0111504528 -o yaml
11:50:48 - MainThread - tests.helpers - INFO - Pod pod-test-0111504528 reached state Pending
11:50:48 - MainThread - ocs_ci.utility.utils - INFO - Executing command: oc -n namespace-test-0111501388 --kubeconfig /home/jijoy/jijoy-ocs-deployment/auth/kubeconfig get Pod pod-test-0111504528 -o yaml
11:50:50 - MainThread - tests.manage.pv_services.test_verify_rbd_based_rwo_pvc - INFO - Running IO on first pod pod-test-0111501948
11:50:50 - MainThread - ocs_ci.utility.utils - INFO - Executing command: oc -n namespace-test-0111501388 --kubeconfig /home/jijoy/jijoy-ocs-deployment/auth/kubeconfig rsh pod-test-0111501948 which apt-get
11:50:53 - MainThread - ocs_ci.utility.utils - INFO - Executing command: oc -n namespace-test-0111501388 --kubeconfig /home/jijoy/jijoy-ocs-deployment/auth/kubeconfig rsh pod-test-0111501948 apt-get update
11:50:59 - MainThread - ocs_ci.utility.utils - INFO - Executing command: oc -n namespace-test-0111501388 --kubeconfig /home/jijoy/jijoy-ocs-deployment/auth/kubeconfig rsh pod-test-0111501948 apt-get -y install fio
11:51:11 - MainThread - ocs_ci.utility.utils - WARNING - Command warning:: debconf: delaying package configuration, since apt-utils is not installed
11:51:11 - ThreadPoolExecutor-1_0 - ocs_ci.utility.workloads.fio - INFO - Running cmd: fio --name=fio-rand-readwrite --filename=/var/lib/www/html/fio-rand-readwrite --readwrite=randrw --bs=4K --direct=1 --numjobs=1 --time_based=1 --runtime=60 --size=1G --iodepth=4 --invalidate=1 --fsync_on_close=1 --rwmixread=75 --ioengine=libaio --output-format=json
11:51:11 - MainThread - ocs_ci.ocs.workload - INFO - Done submitting..
11:51:11 - ThreadPoolExecutor-1_0 - ocs_ci.utility.utils - INFO - Executing command: oc -n namespace-test-0111501388 --kubeconfig /home/jijoy/jijoy-ocs-deployment/auth/kubeconfig rsh pod-test-0111501948 fio --name=fio-rand-readwrite --filename=/var/lib/www/html/fio-rand-readwrite --readwrite=randrw --bs=4K --direct=1 --numjobs=1 --time_based=1 --runtime=60 --size=1G --iodepth=4 --invalidate=1 --fsync_on_close=1 --rwmixread=75 --ioengine=libaio --output-format=json
11:51:11 - MainThread - root - INFO - Waiting for IO results from pod pod-test-0111501948
11:52:38 - MainThread - root - INFO - IOPs after FIO:
11:52:38 - MainThread - root - INFO - Read: 1165.127829
11:52:38 - MainThread - root - INFO - Write: 385.987134
11:52:38 - MainThread - ocs_ci.utility.utils - INFO - Executing command: oc -n namespace-test-0111501388 --kubeconfig /home/jijoy/jijoy-ocs-deployment/auth/kubeconfig rsh pod-test-0111501948 df -kh
11:52:41 - MainThread - ocs_ci.utility.utils - INFO - Executing command: oc -n namespace-test-0111501388 --kubeconfig /home/jijoy/jijoy-ocs-deployment/auth/kubeconfig get Pod pod-test-0111504528 -o yaml
11:52:45 - MainThread - ocs_ci.utility.utils - INFO - Executing command: oc -n namespace-test-0111501388 --kubeconfig /home/jijoy/jijoy-ocs-deployment/auth/kubeconfig get Pod pod-test-0111504528 -o yaml
11:52:50 - MainThread - ocs_ci.utility.utils - INFO - Executing command: oc -n namespace-test-0111501388 --kubeconfig /home/jijoy/jijoy-ocs-deployment/auth/kubeconfig get Pod pod-test-0111504528 -o yaml
11:52:54 - MainThread - ocs_ci.utility.utils - INFO - Executing command: oc -n namespace-test-0111501388 --kubeconfig /home/jijoy/jijoy-ocs-deployment/auth/kubeconfig get Pod pod-test-0111504528 -o yaml
11:52:58 - MainThread - ocs_ci.utility.utils - INFO - Executing command: oc -n namespace-test-0111501388 --kubeconfig /home/jijoy/jijoy-ocs-deployment/auth/kubeconfig get Pod pod-test-0111504528 -o yaml
11:53:02 - MainThread - ocs_ci.utility.utils - INFO - Executing command: oc -n namespace-test-0111501388 --kubeconfig /home/jijoy/jijoy-ocs-deployment/auth/kubeconfig get Pod pod-test-0111504528 -o yaml
11:53:06 - MainThread - ocs_ci.utility.utils - INFO - Executing command: oc -n namespace-test-0111501388 --kubeconfig /home/jijoy/jijoy-ocs-deployment/auth/kubeconfig get Pod pod-test-0111504528 -o yaml
11:53:11 - MainThread - ocs_ci.utility.utils - INFO - Executing command: oc -n namespace-test-0111501388 --kubeconfig /home/jijoy/jijoy-ocs-deployment/auth/kubeconfig get Pod pod-test-0111504528 -o yaml
11:53:15 - MainThread - ocs_ci.utility.utils - INFO - Executing command: oc -n namespace-test-0111501388 --kubeconfig /home/jijoy/jijoy-ocs-deployment/auth/kubeconfig get Pod pod-test-0111504528 -o yaml
11:53:19 - MainThread - ocs_ci.utility.utils - INFO - Executing command: oc -n namespace-test-0111501388 --kubeconfig /home/jijoy/jijoy-ocs-deployment/auth/kubeconfig get Pod pod-test-0111504528 -o yaml
11:53:23 - MainThread - ocs_ci.utility.utils - INFO - Executing command: oc -n namespace-test-0111501388 --kubeconfig /home/jijoy/jijoy-ocs-deployment/auth/kubeconfig get Pod pod-test-0111504528 -o yaml
11:53:27 - MainThread - ocs_ci.utility.utils - INFO - Executing command: oc -n namespace-test-0111501388 --kubeconfig /home/jijoy/jijoy-ocs-deployment/auth/kubeconfig get Pod pod-test-0111504528 -o yaml
11:53:32 - MainThread - ocs_ci.utility.utils - INFO - Executing command: oc -n namespace-test-0111501388 --kubeconfig /home/jijoy/jijoy-ocs-deployment/auth/kubeconfig get Pod pod-test-0111504528 -o yaml
11:53:36 - MainThread - ocs_ci.utility.utils - INFO - Executing command: oc -n namespace-test-0111501388 --kubeconfig /home/jijoy/jijoy-ocs-deployment/auth/kubeconfig get Pod pod-test-0111504528 -o yaml
11:53:40 - MainThread - ocs_ci.utility.utils - INFO - Executing command: oc -n namespace-test-0111501388 --kubeconfig /home/jijoy/jijoy-ocs-deployment/auth/kubeconfig get Pod pod-test-0111504528 -o yaml
11:53:41 - MainThread - tests.manage.pv_services.test_verify_rbd_based_rwo_pvc - INFO - Verified: Second pod pod-test-0111504528 is not in Running state
11:53:41 - MainThread - ocs_ci.utility.utils - INFO - Executing command: oc -n namespace-test-0111501388 --kubeconfig /home/jijoy/jijoy-ocs-deployment/auth/kubeconfig delete Pod pod-test-0111501948
11:53:45 - MainThread - ocs_ci.utility.utils - INFO - Executing command: oc -n namespace-test-0111501388 --kubeconfig /home/jijoy/jijoy-ocs-deployment/auth/kubeconfig get Pod pod-test-0111501948 -o yaml
11:53:46 - MainThread - tests.manage.pv_services.test_verify_rbd_based_rwo_pvc - INFO - First pod pod-test-0111501948 is deleted.
11:53:46 - MainThread - ocs_ci.utility.utils - INFO - Executing command: oc -n namespace-test-0111501388 --kubeconfig /home/jijoy/jijoy-ocs-deployment/auth/kubeconfig get Pod pod-test-0111504528 -o yaml
11:53:51 - MainThread - ocs_ci.utility.utils - INFO - Executing command: oc -n namespace-test-0111501388 --kubeconfig /home/jijoy/jijoy-ocs-deployment/auth/kubeconfig get Pod pod-test-0111504528 -o yaml
11:53:55 - MainThread - ocs_ci.utility.utils - INFO - Executing command: oc -n namespace-test-0111501388 --kubeconfig /home/jijoy/jijoy-ocs-deployment/auth/kubeconfig get Pod pod-test-0111504528 -o yaml
11:53:59 - MainThread - ocs_ci.utility.utils - INFO - Executing command: oc -n namespace-test-0111501388 --kubeconfig /home/jijoy/jijoy-ocs-deployment/auth/kubeconfig get Pod pod-test-0111504528 -o yaml
11:54:03 - MainThread - ocs_ci.utility.utils - INFO - Executing command: oc -n namespace-test-0111501388 --kubeconfig /home/jijoy/jijoy-ocs-deployment/auth/kubeconfig get Pod pod-test-0111504528 -o yaml
11:54:07 - MainThread - ocs_ci.utility.utils - INFO - Executing command: oc -n namespace-test-0111501388 --kubeconfig /home/jijoy/jijoy-ocs-deployment/auth/kubeconfig get Pod pod-test-0111504528 -o yaml
11:54:12 - MainThread - ocs_ci.utility.utils - INFO - Executing command: oc -n namespace-test-0111501388 --kubeconfig /home/jijoy/jijoy-ocs-deployment/auth/kubeconfig get Pod pod-test-0111504528 -o yaml
11:54:16 - MainThread - ocs_ci.utility.utils - INFO - Executing command: oc -n namespace-test-0111501388 --kubeconfig /home/jijoy/jijoy-ocs-deployment/auth/kubeconfig get Pod pod-test-0111504528 -o yaml
11:54:20 - MainThread - ocs_ci.utility.utils - INFO - Executing command: oc -n namespace-test-0111501388 --kubeconfig /home/jijoy/jijoy-ocs-deployment/auth/kubeconfig get Pod pod-test-0111504528 -o yaml
11:54:24 - MainThread - ocs_ci.utility.utils - INFO - Executing command: oc -n namespace-test-0111501388 --kubeconfig /home/jijoy/jijoy-ocs-deployment/auth/kubeconfig get Pod pod-test-0111504528 -o yaml
11:54:29 - MainThread - ocs_ci.utility.utils - INFO - Executing command: oc -n namespace-test-0111501388 --kubeconfig /home/jijoy/jijoy-ocs-deployment/auth/kubeconfig get Pod pod-test-0111504528 -o yaml
11:54:33 - MainThread - ocs_ci.utility.utils - INFO - Executing command: oc -n namespace-test-0111501388 --kubeconfig /home/jijoy/jijoy-ocs-deployment/auth/kubeconfig get Pod pod-test-0111504528 -o yaml
11:54:37 - MainThread - ocs_ci.utility.utils - INFO - Executing command: oc -n namespace-test-0111501388 --kubeconfig /home/jijoy/jijoy-ocs-deployment/auth/kubeconfig get Pod pod-test-0111504528 -o yaml
11:54:41 - MainThread - ocs_ci.utility.utils - INFO - Executing command: oc -n namespace-test-0111501388 --kubeconfig /home/jijoy/jijoy-ocs-deployment/auth/kubeconfig get Pod pod-test-0111504528 -o yaml
11:54:46 - MainThread - ocs_ci.utility.utils - INFO - Executing command: oc -n namespace-test-0111501388 --kubeconfig /home/jijoy/jijoy-ocs-deployment/auth/kubeconfig get Pod pod-test-0111504528 -o yaml
11:54:50 - MainThread - ocs_ci.utility.utils - INFO - Executing command: oc -n namespace-test-0111501388 --kubeconfig /home/jijoy/jijoy-ocs-deployment/auth/kubeconfig get Pod pod-test-0111504528 -o yaml
11:54:54 - MainThread - ocs_ci.utility.utils - INFO - Executing command: oc -n namespace-test-0111501388 --kubeconfig /home/jijoy/jijoy-ocs-deployment/auth/kubeconfig get Pod pod-test-0111504528 -o yaml
11:54:58 - MainThread - ocs_ci.utility.utils - INFO - Executing command: oc -n namespace-test-0111501388 --kubeconfig /home/jijoy/jijoy-ocs-deployment/auth/kubeconfig get Pod pod-test-0111504528 -o yaml
11:55:02 - MainThread - ocs_ci.utility.utils - INFO - Executing command: oc -n namespace-test-0111501388 --kubeconfig /home/jijoy/jijoy-ocs-deployment/auth/kubeconfig get Pod pod-test-0111504528 -o yaml
11:55:07 - MainThread - ocs_ci.utility.utils - INFO - Executing command: oc -n namespace-test-0111501388 --kubeconfig /home/jijoy/jijoy-ocs-deployment/auth/kubeconfig get Pod pod-test-0111504528 -o yaml
11:55:08 - MainThread - tests.manage.pv_services.test_verify_rbd_based_rwo_pvc - INFO - Second pod pod-test-0111504528 is in Running state after deleting the first pod.
11:55:08 - MainThread - ocs_ci.utility.utils - INFO - Executing command: oc -n namespace-test-0111501388 --kubeconfig /home/jijoy/jijoy-ocs-deployment/auth/kubeconfig rsh pod-test-0111504528 df -kh
11:55:11 - MainThread - tests.manage.pv_services.test_verify_rbd_based_rwo_pvc - INFO - Running IO on second pod pod-test-0111504528
11:55:11 - MainThread - ocs_ci.utility.utils - INFO - Executing command: oc -n namespace-test-0111501388 --kubeconfig /home/jijoy/jijoy-ocs-deployment/auth/kubeconfig rsh pod-test-0111504528 which apt-get
11:55:14 - MainThread - ocs_ci.utility.utils - INFO - Executing command: oc -n namespace-test-0111501388 --kubeconfig /home/jijoy/jijoy-ocs-deployment/auth/kubeconfig rsh pod-test-0111504528 apt-get update
11:55:21 - MainThread - ocs_ci.utility.utils - INFO - Executing command: oc -n namespace-test-0111501388 --kubeconfig /home/jijoy/jijoy-ocs-deployment/auth/kubeconfig rsh pod-test-0111504528 apt-get -y install fio
11:55:31 - MainThread - ocs_ci.utility.utils - WARNING - Command warning:: debconf: delaying package configuration, since apt-utils is not installed
11:55:31 - ThreadPoolExecutor-2_0 - ocs_ci.utility.workloads.fio - INFO - Running cmd: fio --name=fio-rand-readwrite --filename=/var/lib/www/html/fio-rand-readwrite --readwrite=randrw --bs=4K --direct=1 --numjobs=1 --time_based=1 --runtime=60 --size=1G --iodepth=4 --invalidate=1 --fsync_on_close=1 --rwmixread=75 --ioengine=libaio --output-format=json
11:55:31 - MainThread - ocs_ci.ocs.workload - INFO - Done submitting..
11:55:31 - ThreadPoolExecutor-2_0 - ocs_ci.utility.utils - INFO - Executing command: oc -n namespace-test-0111501388 --kubeconfig /home/jijoy/jijoy-ocs-deployment/auth/kubeconfig rsh pod-test-0111504528 fio --name=fio-rand-readwrite --filename=/var/lib/www/html/fio-rand-readwrite --readwrite=randrw --bs=4K --direct=1 --numjobs=1 --time_based=1 --runtime=60 --size=1G --iodepth=4 --invalidate=1 --fsync_on_close=1 --rwmixread=75 --ioengine=libaio --output-format=json
11:55:31 - MainThread - root - INFO - Waiting for IO results from pod pod-test-0111504528
11:56:37 - MainThread - root - INFO - IOPs after FIO:
11:56:37 - MainThread - root - INFO - Read: 1280.857305
11:56:37 - MainThread - root - INFO - Write: 426.152462
11:56:37 - MainThread - ocs_ci.utility.utils - INFO - Executing command: oc -n namespace-test-0111501388 --kubeconfig /home/jijoy/jijoy-ocs-deployment/auth/kubeconfig delete Pod pod-test-0111504528
11:56:46 - MainThread - ocs_ci.utility.utils - INFO - Executing command: oc -n namespace-test-0111501388 --kubeconfig /home/jijoy/jijoy-ocs-deployment/auth/kubeconfig get Pod pod-test-0111504528 -o yaml
11:56:47 - MainThread - tests.manage.pv_services.test_verify_rbd_based_rwo_pvc - INFO - Second pod pod-test-0111504528 is deleted.
11:56:47 - MainThread - ocs_ci.utility.utils - INFO - Executing command: oc -n namespace-test-0111501388 --kubeconfig /home/jijoy/jijoy-ocs-deployment/auth/kubeconfig get PersistentVolumeClaim pvc-test-0111501720 -o yaml
11:56:49 - MainThread - ocs_ci.utility.utils - INFO - Executing command: oc -n namespace-test-0111501388 --kubeconfig /home/jijoy/jijoy-ocs-deployment/auth/kubeconfig delete PersistentVolumeClaim pvc-test-0111501720
11:56:50 - MainThread - ocs_ci.utility.utils - INFO - Executing command: oc -n namespace-test-0111501388 --kubeconfig /home/jijoy/jijoy-ocs-deployment/auth/kubeconfig get PersistentVolumeClaim pvc-test-0111501720 -o yaml
11:56:51 - MainThread - tests.manage.pv_services.test_verify_rbd_based_rwo_pvc - INFO - PVC pvc-test-0111501720 is deleted.
11:56:51 - MainThread - ocs_ci.utility.utils - INFO - Executing command: oc -n namespace-test-0111501388 --kubeconfig /home/jijoy/jijoy-ocs-deployment/auth/kubeconfig get PersistentVolume 
11:56:53 - MainThread - ocs_ci.utility.utils - INFO - Executing command: oc -n rook-ceph --kubeconfig /home/jijoy/jijoy-ocs-deployment/auth/kubeconfig delete StorageClass storageclass-test-rbd-0111501599
PASSED                                                                                                                                                                                                                                 [100%]
------------------------------------------------------------------------------------------------------------- live log teardown --------------------------------------------------------------------------------------------------------------
11:56:54 - MainThread - ocs_ci.utility.utils - INFO - Executing command: oc -n namespace-test-0111501388 --kubeconfig /home/jijoy/jijoy-ocs-deployment/auth/kubeconfig delete Project namespace-test-0111501388
11:56:56 - MainThread - ocs_ci.utility.utils - INFO - Executing command: oc -n rook-ceph --kubeconfig /home/jijoy/jijoy-ocs-deployment/auth/kubeconfig delete CephBlockPool cbp-test-0111500672
11:56:58 - MainThread - ocs_ci.utility.utils - INFO - Executing command: oc -n rook-ceph --kubeconfig /home/jijoy/jijoy-ocs-deployment/auth/kubeconfig delete Secret secret-test-rbd-0111500557
11:56:59 - MainThread - ocs_ci.utility.utils - INFO - Executing command: oc -n rook-ceph --kubeconfig /home/jijoy/jijoy-ocs-deployment/auth/kubeconfig get Secret secret-test-rbd-0111500557 -o yaml
11:57:01 - MainThread - root - INFO - Error during execution of command: ['oc', '-n', 'rook-ceph', '--kubeconfig', '/home/jijoy/jijoy-ocs-deployment/auth/kubeconfig', 'get', 'Secret', 'secret-test-rbd-0111500557', '-o', 'yaml'].
                                      Error is Error from server (NotFound): secrets "secret-test-rbd-0111500557" not found
11:57:01 - Dummy-5 - ocs_ci.utility.utils - INFO - Executing command: oc --kubeconfig /home/jijoy/jijoy-ocs-deployment/auth/kubeconfig get Pod -A -o yaml
11:57:01 - Dummy-9 - ocs_ci.utility.utils - INFO - Executing command: oc --kubeconfig /home/jijoy/jijoy-ocs-deployment/auth/kubeconfig get StorageClass -A -o yaml
11:57:01 - Dummy-10 - ocs_ci.utility.utils - INFO - Executing command: oc --kubeconfig /home/jijoy/jijoy-ocs-deployment/auth/kubeconfig get CephFileSystem -A -o yaml
11:57:01 - Dummy-11 - ocs_ci.utility.utils - INFO - Executing command: oc --kubeconfig /home/jijoy/jijoy-ocs-deployment/auth/kubeconfig get CephBlockPool -A -o yaml
11:57:01 - Dummy-12 - ocs_ci.utility.utils - INFO - Executing command: oc --kubeconfig /home/jijoy/jijoy-ocs-deployment/auth/kubeconfig get PersistentVolume -A -o yaml
11:57:01 - Dummy-8 - ocs_ci.utility.utils - INFO - Executing command: oc --kubeconfig /home/jijoy/jijoy-ocs-deployment/auth/kubeconfig get PersistentVolumeClaim -A -o yaml
11:57:01 - Dummy-13 - ocs_ci.utility.utils - INFO - Executing command: oc --kubeconfig /home/jijoy/jijoy-ocs-deployment/auth/kubeconfig get Secret -A -o yaml
11:57:01 - Dummy-4 - ocs_ci.utility.utils - INFO - Executing command: oc --kubeconfig /home/jijoy/jijoy-ocs-deployment/auth/kubeconfig get Namespace -A -o yaml


========================================================================================================= 1 passed in 498.87 seconds =========================================================================================================
```

File: tests/manage/pv_services/test_verify_rbd_based_rwo_pvc.py [hunk 1]
```diff
@@ -274,10 +275,14 @@ def test_rbd_based_rwo_pvc(self, reclaim_policy):
 
         if reclaim_policy == "Delete":
             # Verify PV is deleted
-            pv_info = pv_obj.get(out_yaml_format=False)
-            if pv_info:
-                assert not (pv_name in pv_info), (
-                    f"PV {pv_name} exists after deleting PVC {pvc_obj.name}"
+            for pv_info in TimeoutSampler(
+                    30, 2, pv_obj.get, out_yaml_format=False
```
Comment (RazTamir) [inline tests/manage/pv_services/test_verify_rbd_based_rwo_pvc.py]: this will throw CommandFailed if the PV is gone right?

Comment (jilju) [reply tests/manage/pv_services/test_verify_rbd_based_rwo_pvc.py]: No, it is not getting a particular resource. 
Command will be :
oc -n namespace-test-0111501388 --kubeconfig /home/jijoy/jijoy-ocs-deployment/auth/kubeconfig get PersistentVolume

Comment (RazTamir) [reply tests/manage/pv_services/test_verify_rbd_based_rwo_pvc.py]: Thanks

**Pull request merged**